### ISSUE TITLE
SDDM lock and Sway window manager config

### ISSUE THREAD

**Issue by wakatara** [sddm/sddm#1462]
Hiya SDDM devs!

## TLDR 
I would like SDDM to also act both as my greeter and lockscreen on Arch linux with Sway. Having trouble figuring out how to config this. I have SDDM greeter working and themed for sway wm and wayland for login, just not lock. 

## Longer Tale

I have a freshly installed Arch (2021-10-01 iso) and am using Sway WM on Wayland. I have SDDM installed and working fine as a greeter with the (fantastic) aerial-sddm theme for this. 

I would *like* to use it as the lockscreen to kick in automatically from sway but am honestly having trouble figuring out how to do this. I saw chat of a --lock subcommand searching github history and realize that making a greeter work with the wm can be tricky due to security concerns and making the wm aware of it, but has anyone successfully done this? 

My google-fu failed to find a "yes, you can do it like this "*or* a "no, this is not possible,  shuffle on, so figure I'd better ask direct. 

Would love some help. Love the aerial login screen and it would be epic as a lockscreen (and match the Aerial screensaver I use on my macbook air which I am hoping to replace with Arch as daily driver), but am quite stumped from the docs and such how I might attempt this or if it's even possible with current state of wm. (note the gdm3 lockscreen kicks in automatically at time with sway so the wm does seem to be display manager aware.).

thanks for any help on this! (and if I missed a critical post or obvious config line somewhere, apologies).
